pkg/reconciler/broker/broker.go

```diff
@@ -500,6 +723,34 @@ func makeSubscription(b *v1alpha1.Broker, c *v1alpha1.Channel, svc *corev1.Servi
 	}
 }
 
+// makeSubscriptionCRD returns a placeholder subscription for broker 'b', channelable 'c', and service 'svc'.
+func makeSubscriptionCRD(b *v1alpha1.Broker, c *duckv1alpha1.Channelable, svc *corev1.Service) *v1alpha1.Subscription {
```

Comment:
this should be in the resources dir.

Reply:
Yeah, left it originally in the same file, since makeSubscription was there. Moved.